skipper.go

```diff
@@ -667,6 +667,10 @@ type Options struct {
 	// for a route when it's available (e.g. for RouteGroups)
 	OpenTracingBackendNameTag bool
 
+	// Tracer allows pre-created tracer to be passed on to skipper. Providing the tracer
+	// overrides options provided under OpenTracing property.
+	Tracer ot.Tracer
```

Comment:
@lucastt fyi, let's talk tomorrow about it

Reply:
Same as the other comment, try to be specific with the naming `openTracingTracer` or something like that. Something that specifies that this is OpenTracing, not OpenTelemetry, so when we migrate to OpenTelemetry we support you getting one instance or the other.